RHEL6_7/selinux/RestoreConfig/postupgrade.d/restoreConfig.sh

```diff
@@ -2,7 +2,25 @@
 
 SEMANAGE_EXPORT_FILE=semanage_export
 
+semanage_syntax_update() {
+
+    ftype_opts=( '-f' '-ftype')
+    bool_opts=( '-1' '-0' '--on' '--off' )
+
+    for opt in ${bool_opts[@]}
+    do
+        sed -i -r 's/'"$opt"'/-m '"$opt"'/g' "$1"
+    done
+
+    for opt in ${ftype_opts[@]}
```

Comment:
This should be `"${ftype_opts[@]}"`.  Somebody might add space (or other thing) in the array and your code would break silently.  (Same holds for `bool_opts`.)